spec/features/articles_edited_spec.rb

```diff
@@ -1,23 +1,164 @@
 # frozen_string_literal: true
-
 require 'rails_helper'
 
+slug = 'foo/bar'
+
 describe 'Articles Edited view', type: :feature, js: true do
-  let(:course) { create(:course, start: '2017-01-01', end: '2020-01-01') }
   let(:user) { create(:user, username: 'Ragesoss') }
-  let(:article) { create(:article, title: 'Nancy_Tuana') }
+  let(:home_wiki) { Wiki.get_or_create language: 'en', project: 'wikipedia' }
+  let(:es_wiktionary) { create(:wiki, language: 'es', project: 'wiktionary') }
+  let(:fr_wikidata) { create(:wiki, language: 'fr', project: 'wikidata') }
```

Comment:
There is no `fr.wikidata`. Wikidata is multilingual project, and within the dashboard, the language attribute for it will be `null`. See `wiki.rb#81`.